time_tracker_api/api.py

```diff
@@ -34,6 +34,19 @@ def remove_required_constraint(model: Model):
     return result
 
 
+def add_update_last_entry_if_overlap(time_entry_model: Model):
+    time_entry_flag = {
+        'update_last_entry_if_overlap': fields.Boolean(
+            title='Update last entry if overlap',
+            required=False,
+            description='Flag that indicates if the last time entry is updated',
+            example=True,
+        )
+    }
+    new_model = time_entry_model.clone('TimeEntryInput', time_entry_flag)
```

Comment:
It is neccesary to use the `clone` method?. Because if not,the method woudl be simpler.

Reply:
It is necessary, the idea was used extended method, but it was deprecated. 
I didn't see the way to add a new field without changing the TimeEntry model defined in time_entries_namespace